common/script/index.coffee

```diff
@@ -755,7 +755,7 @@ api.wrap = (user, main=true) ->
         if user.balance < 1
           return cb? {code:401,message: i18n.t('notEnoughGems', req.language)}
         else
-          user.balance--
+          user.balance -= 1
```

Comment:
I'm curious: why this change? I'm sure you have a good reason, I'd just like to know what it is, for future use.